pkg/k8s/service_cache.go

```diff
@@ -106,6 +106,8 @@ func NewServiceCache(nodeAddressing datapath.NodeAddressing) ServiceCache {
 }
 
 // GetServiceIP returns a random L3n4Addr that is backing the given Service ID.
+// The returned IP is with external scope since it's string representation might
```

Comment:
```suggestion
// The returned IP is with external scope since its string representation might
```
(If you find other things to address; don't respin for that.)

Reply:
Ok, I'll wait for the current CI to go through, and if all green, then fix the comment and wait only for travis.